modules/ROOT/pages/deploying-to-cloudhub.adoc

```diff
@@ -9,57 +9,96 @@ image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies", ti
+
+[TIP]
+With the CloudHub dedicated load balancer, you can then optionally implement xref:cloudhub-dedicated-load-balancer.adoc#mapping-rules[mapping rules] to set your application's public URL to any other path you choose, as long as you own the domain.
+
+== Deployment Target
+
+If you already have servers registered to the Anypoint Platform, you see a *Deployment Target* field in the *Deploy Application* page. Use it to specify where you want to deploy your application, in this case, CloudHub.
+
+For more information about deploying to on-premises servers, server groups, and clusters, see xref:deploying-to-your-own-servers.adoc[Deploying to Your Own Servers].
+
+
+== Deploy an Application to CloudHub from Studio
 
 You can deploy your applications to CloudHub from Studio. This is helpful if you're still developing the application and want to deploy it often to an online test environment.
 
 . In Studio, open your application as a Mule Project.
-
 . In *Package Explorer*, right-click the project node, then select *Anypoint Platform*, then *Deploy to CloudHub*.
 . If this is your first time deploying from Studio, a popup dialog box asks you to provide your login credentials for Anypoint Platform. Studio stores your credentials and uses them automatically the next time you deploy to CloudHub.
-
 +
 [TIP]
 You can manage these credentials through the Studio *Preferences* menu, in *Anypoint Studio* > *Authentication*.
-
 . After you sign in, the *Deploy Application* dialog opens.
 +
 image::deployappfirstscreen.png[DeployAppFirstScreen]
-
++
 [NOTE]
 --
 You cannot deploy an application from Studio to the CloudHub *Design* environment. The *Design* environment is specific to applications created in Anypoint Platform Design Center.
 --
 
-=== Deploy from Runtime Manager
+== Deploy an Application to CloudHub from Runtime Manager
 
-. Sign on to https://anypoint.mulesoft.com[Anypoint Platform] or click *Sign up* on the sign on page.
-. Hover your mouse over the *Runtime Manager* icon and click on it, or click *Open*.
-+
-image::deploying-to-cloudhub-5f383.png[]
-+
-. In the *Applications* tab, click *Deploy application*.
+. In Anypoint Platform, go to *Management Center > Runtime Manager*. 
+. In the *Applications* tab, click *Deploy application*. 
+. In the *Deploy Application* page: +
+* Enter an *Application Name* +
+[NOTE]
+An application with the same name cannot be deployed in two environments at the same time.
+* In the *Deployment Target* drop-down list, select *CloudHub*.
+* Click *Choose File*, then select either:
+** *Import file from Exchange*
+** *Upload file* 
 +
-// image::deploying-to-cloudhub-3dcbb.png[]
+[NOTE]
+The application file size limit is 200 MB.
 +
-The Deploy Application page opens:
+You can also click *Get from sandbox* to copy a file from a non-production environment into your current environment. (If the *Get from sandbox* button does not appear, you first need to create a non-production environment.)
```

Comment:
Can we tell the customer "what & why" before "how" (so if they aren't interested in the "why" they don't have to read any further? This isn't a formal style yet. Is this the most important thing to tell them or can this be further down in the instruction? For example, line 161?